backend/corpora/common/entities/collection.py

```diff
@@ -58,31 +57,28 @@ def create(
 
     @classmethod
     def get_collection(
-        cls, session, collection_uuid, visibility=CollectionVisibility.PUBLIC.name, include_tombstones=False
-    ):
+        cls,
+        session,
+        collection_uuid: str,
+        visibility: str = CollectionVisibility.PUBLIC.name,
+        include_tombstones: bool = False,
+        owner: typing.Optional[str] = None,
```

Comment:
@tihuan owner is an optional parameter.

Reply:
oh!! Thank you 🙏  I didn't scroll up high enough to see this 😆 My bad!